doc/tutorials/SIR.ipynb

```diff
@@ -507,14 +507,49 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "outputPrepend",
```

Comment:
What is this for?

Reply:
This is again something that vscode adds and makes the docs crash.